styles/components/_trading-card-num-results.scss

```diff
@@ -0,0 +1,4 @@
+.cmp-trading-card-num-results {
+  font-size: font-size("lg");
+  margin-bottom: 2rem;
```

Comment:
I think the font is slightly off, looks like Figma is using a different variant of the base font.

```
  font-family: font-family("sans-extended");
```

Reply:
Good catch! I'm still learning my way around Figma.